src/index.js

```diff
@@ -62,9 +63,27 @@ async function main() {
       addedFiles = files.data ? files.data.filter(file => file.status === 'added').map(file => file.filename) : [];
     }
 
-    // Read the json summary files for base and branch coverage
-    const coverageReportNew = JSON.parse(fs.readFileSync(branchCoverageReportPath).toString());
-    const coverageReportOld = JSON.parse(fs.readFileSync(baseCoverageReportPath).toString());
+    const coverageFileType = branchCoverageReportPath.match(/\.([^\.]+$)/)[1];
+    let coverageReportNew;
+    let coverageReportOld;
+    if (coverageFileType === 'json') {
+      // Read the json summary files for base and branch coverage
+      coverageReportNew = JSON.parse(fs.readFileSync(branchCoverageReportPath).toString());
+      coverageReportOld = JSON.parse(fs.readFileSync(baseCoverageReportPath).toString());
+    } else if (coverageFileType === 'xml') {
+      coberturaParseContent(fs.readFileSync(branchCoverageReportPath).toString())
```

Comment:
```suggestion
parseContent(fileContent, type)
```
type = ['jest', 'cobertura', 'android', 'roku']